### ISSUE TITLE
Option for Find Results to open in a new window

### ISSUE THREAD

**Issue by jacohn16** [atom/find-and-replace#989]
I came from using TextMate and the Project Find would open as a new window.  I think this would be a good thing for Atom as well, because I tend to find all the windows (Project files, file tabs, git tab, index tab, etc...) all get vertically scrunched in the same window.  

How about an option to open the Find/Replace and/or Find Results in a separate window.  

Thoughts?

**Comment by Daijobou** [comment]
Right click on tab and use "open in new window"? So its optional for everyone. You can split the tab too.

**Comment by moonlight16** [comment]
@Daijobou 
Yes, that's comparable to what TextMate allows you to do.  Splitting the tab really just compounds the issue as it squeezes in another window within the single larger window.  So yeah, I think an option when you right-click would be best.  It already has options like, "Close Tab", "Close All Tabs", etc..  So another option like "Move/Open in New Window" would fit well.

**Comment by Daijobou** [comment]
I not understand. That option exists already in popup :)

![grafik](https://user-images.githubusercontent.com/260343/37541333-8b6c022a-295a-11e8-83cc-b8a35d456fb0.png)

**Comment by roguenorman** [comment]
When I right click on a file's tab, it shows this option. I then click ctrl + shift + F and do a search. Then right click on the tab labeled "Project find results" it does not have the option to open in a new window.

![image](https://user-images.githubusercontent.com/20261531/37636624-3e11e8f0-2c57-11e8-9bb9-39388ea69961.png)

**Comment by Daijobou** [comment]
Oh, you are right. :)

In this case, open first in treeview a new window and search in this new atom instance. I see no difference open first a new window before search or after search. 

I use 3-6 instances of different subfolder from my project. Its much faster to select editor in taskbar as scroll and collapse or expanding tons of folders. ^^
